src/main/java/com/wheretoeat/service/IRestaurantService.java

```diff
@@ -11,11 +11,11 @@
 	 * @param type - Takes a food type (category) enum 
 	 * @return will return a restaurant object based on the food type selected.
 	 */
-	RestaurantDTO fetchByFoodType(FoodType type);
+	RestaurantDTO fetchByFoodType(FoodType type) throws Exception;
```

Comment:
Should handle Exception

Reply:
Does it actually throw an exception?  If not, leave this off.

Reply:
It should. In my experience, any services that handle querying the database always have the potential to throw DB Exception (Ex: Connection Exception).